src/shapes/Object/defaultValues.ts

```diff
@@ -36,75 +36,48 @@ export const cacheProperties = [
 ];
 
 export const fabricObjectDefaultValues = {
-  originX: LEFT,
-  originY: TOP,
+  // coming from ObjectOrigin
   top: 0,
   left: 0,
   width: 0,
   height: 0,
-  scaleX: 1,
-  scaleY: 1,
   flipX: false,
   flipY: false,
-  opacity: 1,
-  angle: 0,
+  scaleX: 1,
+  scaleY: 1,
   skewX: 0,
   skewY: 0,
-  cornerSize: 13,
-  touchCornerSize: 24,
-  transparentCorners: true,
-  hoverCursor: null,
-  moveCursor: null,
+  originX: LEFT,
+  originY: TOP,
+  angle: 0,
```

Comment:
IMO opinion it makes sense to have the transform listed in the order iti s applied


translate
rotation
scaling, flipping
skewing

Reply:
so sounds like angle should be below hieght, scale's above flip's?
Should I also add a comment like // transforms apply translation, then rotation, scaling, flipping, skewing?
Would you like the comments about the super classes removed to allow for where transforms should list?

Reply:
> Should I also add a comment like // transforms apply translation, then rotation, scaling, flipping, skewing?

No because we have a comment about it I think `composeMatrix` or something. You could point to that method.

> Would you like the comments about the super classes removed to allow for where transforms should list?

Which?

Reply:
> Would you like the comments about the super classes removed to allow for where transforms should list?

I see know what you mean. Yes I think they do not belong here and keep the legacy design we don't want stuck in our code and mind

Reply:
changed comment to reflect composeMatrix, pulled out comments about superclasses and minor reorder.